recipes-kernel/linux/files/5.x-kernel/0009-dts-add-the-usb3.0-support.patch

```diff
@@ -0,0 +1,48 @@
+From fd343f8a90b0636bb0c7043cc0ca1e3aa74e1893 Mon Sep 17 00:00:00 2001
+From: Chao Zeng <chao.zeng@siemens.com>
+Date: Fri, 19 Mar 2021 14:40:24 +0800
+Subject: [PATCH 09/14] dts: add the usb3.0 support
+
+enable serdes0 and add the clock configuration
+
```

Comment:
This is only targeting PG2, both variants, right?

Reply:
yes,for both PG2 variants.